GraphStructure/Graph.java

```diff
@@ -0,0 +1,103 @@
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.Iterator;
+
+public class Graph{
+	public ArrayList<Vertex> vertexes;
+	public ArrayList<Boolean> visitMark;
```

Comment:
Should this be here? I donot think it gonna be used anywhere other than graph traversal.

Reply:
Hi,should I add a variable like "isVisit " into the class "Vertex",which can be used in graph traversal.

Reply:
Again, this only confines to graph traversal. Using a local hashtable in your traversal algorithm to indicate the visiting status should suffice.

Reply:
Hi, I think Set is also proper here, the vertex can be treated as visited if it is in the set,otherwise,it is treated as unvisited.

Reply:
you are right.